### ISSUE TITLE
Daily alarms?

### ISSUE THREAD

**Issue by sumyungguy** [nriley/Pester#19]
Hi - First, thanks for Pester, it's great!!

One thing though that seems pretty basic, I don't see how to make a normal alarm clock, eg. wake-up at 8 am every day. Am I missing something?

**Comment by nriley** [comment]
I agree, it'd be an obvious and useful improvement.  It's on the list (first priority, actually) — unfortunately it's also been in that position for a couple years, as I spend most of my limited time working on Pester just keeping up with OS X updates.

**Comment by sumyungguy** [comment]
Ok, understandable... In the meantime I slapped together a simple Applescript for my "go to bed" alarm. Actually it's cool because I can use it with "banner" style OS X notifications that I don't have to dismiss.

I still use Pester all the time, it's saved me from countless burned dinners and exploding beers in the freezer! So thanks again.
